Dockerfile

```diff
@@ -5,9 +5,11 @@ RUN apk add --no-cache bash
 # Setting working directory.
 WORKDIR /usr/src/app
 
+RUN corepack enable yarn
```

Comment:
This is required in the latest node 20 image since we set the `packageManager` field in `package.json`